charts/outdated/values.yaml

```diff
@@ -30,6 +30,7 @@ ember:
 
 django:
   allowedHosts: "*"
+  watchmanChecks: ~
```

Comment:
Using an array might be cleaner given that Helm will consider that to be falsey as well.
```suggestion
  watchmanChecks: []
```

Reply:
oops